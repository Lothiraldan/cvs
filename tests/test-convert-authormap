#!/bin/sh

cat >> $HGRCPATH <<EOF
[extensions]
convert=
EOF

# Prepare orig repo
hg init orig
cd orig
echo foo > foo
HGUSER='user name' hg ci -qAm 'foo'
cd ..

# Explicit --authors
cat > authormap.txt <<EOF
user name = Long User Name

# comment
this line is ignored
EOF

hg convert --authors authormap.txt orig new
echo $?
cat new/.hg/authormap

hg -Rnew log
rm -rf new

# Implicit .hg/authormap
hg init new
mv authormap.txt new/.hg/authormap

hg convert orig new
echo $?

hg -Rnew log
