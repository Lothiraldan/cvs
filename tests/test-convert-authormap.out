initializing destination new repository
scanning source...
sorting...
converting...
0 foo
Writing author map file new/.hg/authormap
0
user name=Long User Name
changeset:   0:d89716e88087
tag:         tip
user:        Long User Name
date:        Thu Jan 01 00:00:00 1970 +0000
summary:     foo

scanning source...
sorting...
converting...
0 foo
0
changeset:   0:d89716e88087
tag:         tip
user:        Long User Name
date:        Thu Jan 01 00:00:00 1970 +0000
summary:     foo

